streams/examples/src/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedJob.java

```diff
@@ -124,9 +112,9 @@ public static void main(String[] args) throws Exception {
         serdeProps.put("JsonPOJOClass", RegionCount.class);
         regionCountSerializer.configure(serdeProps, false);
 
-        KStream<String, PageView> views = builder.stream(stringDeserializer, pageViewDeserializer, "streams-pageview-input");
+        KStream<String, PageView> views = builder.stream(Serdes.String(), Serdes.serdeFrom(null, pageViewDeserializer), "streams-pageview-input");
```

Comment:
The `null` arguments in `Serdes.serdeFrom(null, pageViewDeserializer)` and `Serdes.serdeFrom(wPageViewByRegionSerializer, null)` look weird to me -- we had a similar case in previous versions of the code, which we were able to get rid of.  Now nulls are back with unified serdes. :-)

Is there a way we can smoothen this out?  What about introducing some sugar such as:
```java
// replace
Serdes.serdeFrom(null, pageViewDeserializer);

// with
Serdes.partialSerdeFromDeserializer(pageViewDeserializer); // <<< I'd prefer this, it's similar to "partial function" terminology
// or
Serdes.serdeFromDeserializer(pageViewDeserializer);
```